src/components/AddActivity.tsx

```diff
@@ -26,85 +28,90 @@ export default function AddActivity() {
           }
           title="Legg til ny aktivitet"
         />
+
         <CardContent>
           <Typography component={'span'} variant="body2" color="text.secondary">
               <FormControl variant="standard">
                   <Box
-                  component="form"
-                  sx={{'& .MuiTextField-root': { m: 1, width: '100%' },}}
-                  noValidate
-                  autoComplete="off"
+                    component="form"
+                    sx={{'& .MuiTextField-root': { m: 1, width: '100%' },}}
+                    noValidate
+                    autoComplete="off"
                   >
-                  <div>
-                      <TextField
-                      required
-                      id="outlined-required"
-                      label="Tittel på aktivitet"
-                      />
 
-                      <TextField
-                      id="outlined-number"
-                      label="Dato"
-                      type="date"
-                      InputLabelProps={{
-                          shrink: true,
-                      }}
-                      />
+                    <div>
+                        <TextField
+                          required
+                          id="outlined-required"
+                          label="Tittel på aktivitet"
+                        />
+
+                        <TextField
+                          id="outlined-number"
+                          label="Dato"
+                          type="date"
+                          InputLabelProps={{
+                              shrink: true,
+                        }}
```

Comment:
Minor comment: as no prettyfier is applied in your VS code, these closing brackets are misaligned.
They should be aligned to `InputLabelProps`, while now they are aligned to `<TextField` (making it harder to read => harder to see where a block starts and ends)

Reply:
Can you show me how to apply the prettyfier in VS code?

Reply:
Sure! I'll check what setup I have and I'll show you.